packages/nimble-components/src/tabs/styles.ts

```diff
@@ -22,5 +22,6 @@ export const styles = css`
         grid-row: 2;
         grid-column-start: 1;
         grid-column-end: 4;
+        overflow: auto;
```

Comment:
Any thoughts on `overflow` vs `overflow-y`? I could see an argument that Nimble officially supports vertically scrollable content but officially recommends against horizontally scrollable content since horizontal scrollbars are annoying.

Reply:
I don't have a strong feeling about that. I don't personally have negative feelings towards horizontal scrollbars, but if that is a common sentiment, we can make this `overflow-y` instead. Should I change it?

Reply:
My thought process is that cutting off content will signal that the configuration is unsupported and clients should update their content to be narrower. Alternatively, scrolling content might signal that the configuration is supported (when I assert that configuration should be avoided).

If no one else has strong opinions I would vote to change it.

Reply:
+1 that we should not encourage horizontal scrollbars.

I'm a little concerned about doing overflow hidden because I think it's more likely a client will mess-up and i.e. underestimate the size of some dynamic content that is inserted. If the overflow is hidden it could make that content difficult to access or be aware of for end users as there would be no visual affordance that extra content is hidden.

I think I'd rather the end user have an ugly operable visual than something that feels clipped or makes content inaccessible. But the design guidance for clients is that they should make the content fit correctly and only scroll vertically.

Reply:
I'm ok with that approach. Could you add a sentence to the usage docs in storybook that mentions these guidelines on content size?

Reply:
Added a couple sentences.